pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```
@@ -81,6 +81,11 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/teams-for-linux --version | grep -F "${version}" >/dev/null
+  '';
+
```

Should we just use `testers.testVersion` instead if we really want to add this?

```suggestion
  passthru.tests.version = testers.testVersion { package = teams-for-linux; };

```